README.md

```diff
@@ -1 +1,46 @@
-# lucidity-plasma-cash
\ No newline at end of file
+# Lucidity Plasma
+
+Implements a Solidity contract to manage Plasma side-chain on Ethereum.
+
+## Install dependencies with
+
+```
+$ npm install
+```
+
+
+## Run test cases:
+
+```
+# start ganache CLI
+$ ./ganache-cli
+
+$ truffle test
+```
+
+## Details
```

Comment:
Should add link to https://ethresear.ch topic.
I think it's: https://ethresear.ch/t/plasma-cash-plasma-with-much-less-per-user-data-checking/1298